packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  LinkLibrary,
+  thunderstoreLinkProps,
+  ThunderstoreLinkProps,
+} from "@thunderstore/components";
+import NextLink from "next/link";
+import React, { forwardRef } from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps, ThunderstoreLinkProps {
+  queryParams?: string;
+  url: string;
+}
+
+const CyberstormLink = forwardRef<HTMLAnchorElement>(
+  (props: CyberstormLinkProps, ref) => {
+    const { ...rest } = props;
```

Comment:
Why do you think the place where this was copied from uses the `...rest` operator? What does it do here?

Reply:
Well the answer I was looking for is that using the `...rest` in [destructing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) like this means you create a `const rest` that is an exact copy of props. Which is kinda pointless, no?

Reply:
Yea correct, in that line 20 (which is now outdated) it would have been pointless. Good that you pointed out that destructing assignment, I wasn't fully aware how it worked 👍